### DISCUSSION TITLE
[Feature Request] Toggle button for live timing sync

### DISCUSSION THREAD

**Discussion by Sloth-on-meth** [f1multiviewer/issue-tracker#571] [Feature requests] [answered]
hi there!

i like to have the live timing on 0 delay just because i love knowing something is _about to_ happen. would love there to be  a toggleswitch to disable live timing sync, because the video streams keep trying to catch up with 0 delay, which is nearly impossible.

**Comment by LapsTimeOFF** [comment]
Maybe post it in the discussions part

**Comment by jeppeolesen** [comment, marked as answer]
There are now experimental options in MultiViewer that you can use to disable some of the sync features.

![Screenshot 2025-03-23 at 01 38 37](https://github.com/user-attachments/assets/70fe4695-1839-4c56-9306-90ca0280a45e)
